### PR TITLE
Update example to use LoopBack 3.x

### PR DESCRIPTION
**Pull request by 0candy**
Connect to strongloop/loopback#2772

### CODE REVIEW

**Comment by 0candy** [comment]
@bajtos @superkhau PTAL

**Comment by bajtos** [comment]
@0candy could you please add [server/middleware.development.json](https://github.com/strongloop/loopback-workspace/blob/27572451152fdcadf15de6f3ab4b11316a3f27a0/templates/projects/empty-server/files/server/middleware.development.json)? Troubleshooting failed requests is cumbersome otherwise.

**Comment by 0candy** [comment]
@bajtos Added. PTAL

File: server/datasources.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "db": {
-    "connector": "memory",
-    "defaultForType": "db"
+    "name": "db",
```
Comment (bajtos) [inline server/datasources.json]: FWIW, this looks like a bug of loopback-workspace to me. The `name` property should not be included in the JSON output, as the name is already stored as the key (top-level property name).

No need to fix it here though.

**Comment by 0candy** [comment]
CI failures are due to PhantomJS on Windows. There is already an issue created for that.

**Pull request merged**